.github/workflows/make_release.yml

```diff
@@ -30,9 +30,42 @@ env:
   NPM_TAG: ""
 
 jobs:
+  package:
+    runs-on: ubuntu-latest
+    outputs:
+      hash: ${{ steps.hash.outputs.hash }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+        fetch-depth: 0
+    - name: Prepare package
+      run: |
+        cargo package -p tfhe
+    - name: generate hash
+      id: hash
+      run: cd target/package && echo "hash=$(sha256sum ./*.crate | base64 -w0)" >> "${GITHUB_OUTPUT}"
```

Comment:
I'm confused, we create a package here, hash it, and then we re create a package later in the next job to publish ? 🤔